test/e2e/Makefile

```diff
@@ -44,7 +57,7 @@ e2e-cli-plugin-compatibility-test:
 	else \
 		export TANZU_CLI_E2E_TEST_CENTRAL_REPO_URL=$(TANZU_CLI_E2E_TEST_CENTRAL_REPO_URL) ; \
 		export TANZU_CLI_PLUGIN_DISCOVERY_IMAGE_SIGNATURE_VERIFICATION_SKIP_LIST=$(TANZU_CLI_E2E_TEST_CENTRAL_REPO_URL) ; \
-		export TANZU_CLI_CEIP_OPT_IN_PROMPT_ANSWER="Yes" ; \
+		export TANZU_CLI_CEIP_OPT_IN_PROMPT_ANSWER="No" ; \
```

Comment:
> I assume you changed this because we don't want these tests to affect our telemetry statistics?
Yes, that's correct.

Updated the other 4 instances too.

Yes, we should add a suggestion for plugin devs regarding setting CEIP to "No" so as to not affect the telemetry statistics. We can do it as part of telemetry implementation

Reply:
> Yes, we should add a suggestion for plugin devs regarding setting CEIP to "No" so as to not affect the telemetry statistics. We can do it as part of telemetry implementation

Good for me, thanks.